README.md

```diff
@@ -1,13 +1,30 @@
 # Migration Utilities [![Build Status](https://travis-ci.org/fcrepo4-exts/migration-utils.png?branch=master)](https://travis-ci.org/fcrepo4-exts/migration-utils)
 
-A framework to support migration of data from Fedora 3 to Fedora 4 repositories.
+A framework to support migration of data from Fedora 3 to Fedora 4, 5, or 6 repositories
 
 ## Overview
 
-The main class (`org.fcrepo.migration.Migrator`) iterates over all of the fedora objects in a configured source (`org.fcrepo.migration.ObjectSource`) and handles them using the configured handler (`org.fcrepo.migration.StreamingFedoraObjectHandler`). The configuration is entirely contained within a Spring XML configuration file in [`src/main/resources/spring/migration-bean.xml`](https://github.com/fcrepo4-exts/migration-utils/blob/master/src/main/resources/spring/migration-bean.xml).
+This utility iterates the foxml files of a fedora 2 or 3 repository, and populates a fedora 4, 5, or 6 repository.
+
+For migrations to Fedora 4 and 5, the utility populates the repository via its APIs.  You will need a running fedora 4 or fedora 5 repository to perform the migration.
+The utility will perform various mapping operations in order to fit the fedora 2/3 model onto LDP as supported in Fedora 4 and 5.  In particular:
+
+* All RDF URIs will be re-mapped.  Fedora 2 and 3 use `info:fedora/` URIs in `RELS-EXT` and `RELS-INT`.  THe migration utility will re-write these URIs into resolvable `http://` URIs that point to the corresponding resources in the fedora 4 or 5 repository
+* FOXML object properties will be expressed in terms of RDF according to a mapping
```

Comment:
"according to the mapping [defined in ${migration.mapping.file}]". See example "[custom-mapping.properties](https://github.com/fcrepo4-exts/migration-utils/blob/master/src/main/resources/custom-mapping.properties)"